lms/djangoapps/certificates/views/webview.py

```diff
@@ -155,27 +104,96 @@ def _update_certificate_context(context, course, user, user_certificate):
     )
 
     if partner_long_name:
-        context['accomplishment_copy_course_description'] = _('a course of study offered by {partner_short_name}, an '
-                                                              'online learning initiative of {partner_long_name} '
-                                                              'through {platform_name}.').format(
-            partner_short_name=partner_short_name,
-            partner_long_name=partner_long_name,
-            platform_name=platform_name
-        )
+        context['accomplishment_copy_course_description'] = _(
```

Comment:
@ziafazal as we discussed offline, multi-line translated strings need to start on the same line as the opening parenthesis in order for translators' comments to carry through correctly.   Also we should try to include comments where they are missing, if possible.  (such as for this particular string)